packages/docusaurus-1.x/lib/copy-examples.js

```diff
@@ -15,6 +15,26 @@ const path = require('path');
 
 const CWD = process.cwd();
 
+const toHex = color => {
+  const hex = color.toString(16);
+  return hex.length === 1 ? `0${hex}` : hex;
+};
+
+const colorScheme = () => {
+  let primary = '#';
+  let secondary = '#';
+  for (let i = 0; i < 3; i++) {
+    const color = Math.floor(Math.random() * 176);
```

Comment:
Where did this number come from? Is it so that the color wouldn't be too bright? If so, please add a comment.

Reply:
Added the comment! Thats exactly right :) 175 is a bit arbitrary though.